buf.gen.yaml

```diff
@@ -0,0 +1,4 @@
+version: v1
+plugins:
+  - remote: buf.build/stephenh/plugins/ts-proto
```

Comment:
Okay, having looked at this I think I understand what you're doing now in generate.yaml. It seems to me that you could also equally achieve this by adding https://github.com/stephenh/ts-proto or https://www.npmjs.com/package/ts-protoc-gen as NPM dependencies, and then letting `npx semantic-release` run as usual. Am I right?

Reply:
I'm not sure it will manage to catch the changes on https://buf.build/cheqd/cheqd-proto/docs/main that way.
I was thinking that we can trigger this workflow from the cheqd-node repository each time there is a new change in protobuf definitions.

Reply:
Protobuf definitions a consensus breaking change that doesn't happen too often. So for example if it's `proto/did/v2` now, the ONLY update once on main will require next version to be `proto/did/v3`. So you don't need to dispatch it for every commit/merge on cheqd-node main.

What I don't understand is how you've got v2 protobufs yet, since I don't think we're publishing these to buf.build yet, so I'm not sure how pulling it down from buf.build is working. IIRC I think _both_ main and develop in cheqd-node publish to the same "main" in buf.build, so it's actually co-mingled. Maybe we need to chat this through to see we're pulling down the right thing to build.

I'd suggest making this manual, on workflow dispatch, for syncing upstream protobuf changes.

And keep the normal build/release/deploy for NPM release (which can also handle dependabot etc).